Sources/ApolloCodegenLib/ApolloCodegenConfiguration.swift

```diff
@@ -1370,3 +1381,36 @@ extension ApolloCodegenConfiguration.OutputOptions {
     }
   }
 }
+
+private struct AnyCodingKey: CodingKey {
+    var stringValue: String
+
+    init?(stringValue: String) {
+        self.stringValue = stringValue
+    }
+
+    var intValue: Int?
+
+    init?(intValue: Int) {
```

Comment:
The `intValue` stuff here is throwing me off. This seems like it can't actually be used properly for validating  the coding keys. Am I missing the reason for this?

Reply:
Truthfully, I borrowed the `AnyCodingKey` from elsewhere; and it's implemented for int-keyed dictionaries. However, that doesn't really apply to decoding from JSON, so I'll remove it.

Reply:
It's a requirement of the protocol
<img width="1177" alt="Screenshot 2023-07-18 at 8 44 43 PM" src="https://github.com/apollographql/apollo-ios/assets/3388381/a880b75f-d223-43b7-adbd-c5cb76871080">

Reply:
Ahhh odd. Okay, thanks for explaining :)